### PR TITLE
Created the calculator design

### PR DESCRIPTION
**Pull request by ticoniq**
### **Created the calculator design**

Added a React component that holds the core functionality: a calculator.

Deleted all the boilerplate from Create React App 
Create a new directory called components.
Create a React component that matches the design:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import '../assets/App.css';
+
+function Calculator() {
+  return (
+    <div className="container">
+      <div className="calculator">
+        <div className="keypad">
+          <div className="display">0</div>
+          <div className="row">
+            <button type="button" className="key">AC</button>
```
Comment (amejid) [inline src/components/Calculator.js:11]: - [ ] You did a fantastic job with the design. Let's move some code around to achieve the project requirements. We should have at least one child component inside this component. Kindly create a component and use it inside this one. 😎️

**Project requirement**
Screenshot from 2023-07-05 08-45-15

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @ticoniq, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (ticoniq) [reply src/components/Calculator.js:11]: Thanks for the feedback, please explain

Comment (ticoniq) [reply src/components/Calculator.js:11]: Because i already create a component call calculator

**Review by Ibizugbe — COMMENTED** [review]
Hi @ticoniq 👋👋, 

INVALID


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your project doesn't have any change since the last review😉. I will expantiate on what the previous reviewer wants. As pointed out by the previous reviewer, It is a requirement that the `Calculator.js` component should have atleast one child component as this makes our project easier to understand and maintain by achieving the principle of seperation of concerns😉. Your calculator component can further be split into the `button` and `screen` component😉 and this will serve as the child components of the `calcu;ator` component😉. Just make sure your `calculator` component has atleast one child component to fulfill the requirement👍.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ticoniq ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Added child component ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**